kotlin-spark-api/src/main/kotlin/org/jetbrains/kotlinx/spark/api/Encoding.kt

```diff
@@ -119,6 +120,7 @@ private val knownDataTypes: Map<KClass<out Any>, DataType> = mapOf(
     ByteArray::class to DataTypes.BinaryType,
     Decimal::class to DecimalType.SYSTEM_DEFAULT(),
     BigDecimal::class to DecimalType.SYSTEM_DEFAULT(),
+    BigInteger::class to DecimalType.SYSTEM_DEFAULT(),
```

Comment:
Does it work for all the Spark versions available?

Reply:
As far as I can see, the DecimalType is present since 1.3.0, so then yes